flo_telepresence/src/display_cv.py

```diff
@@ -213,6 +218,7 @@ def __run_display(self):
                 except Queue.Empty:
                     empty = True
             if img is not None:
+                print(img.shape)
```

Comment:
don't want to see print statements in final code

Reply:
I did not add this line of code, It was a part of the original code, I can remove it if that's in line with the final code formatting.

Reply:
huh, wonder why it is highlighted green. Yeah, please remove it.